dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -157,11 +157,10 @@ setup(){
             popd
         fi
         echo "Clean-up existing OSD access keys.."
-        DELETE_KEYS=$(aws iam list-access-keys --user-name OsdCcsAdmin | awk '{print$2}')
-        for i in $DELETE_KEYS
-        do
-            aws iam delete-access-key --access-key-id $i --user-name OsdCcsAdmin
-        done
+        aws_key=$(aws iam list-access-keys --user-name OsdCcsAdmin --output text --query 'AccessKeyMetadata[1].AccessKeyId')
+        if [[ $aws_key != None ]]; then
+            aws iam delete-access-key --user-name OsdCcsAdmin --access-key-id $aws_key
+        fi
```

Comment:
I'd recomend we do something more like this so we delete the oldest key (the one at index 0) instead
```
aws_key=(`aws iam list-access-keys --user-name OsdCcsAdmin --output text --query 'AccessKeyMetadata[*].AccessKeyId'`)
if [[ ${#aws_key[@] -eq 2 ]]; then
  del aws_key[0]
fi
````

Reply:
Agreed, we should delete the last active key when both are active.

Reply:
len_aws_key should be ${len_aws_key}